content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: blog
+title:  Airflow on Kubernetes (Part 1)\: A Different Kind of Operator 
```

Comment:
Title will work with the semi-colon if formatted as: 'Airflow on Kubernetes (Part 1): A Different Kind of Operator '